hasura/graphql-engine
# cli: load assets from server if cdn is disabled (close #3382)

### Description
Currently, the console run via the CLI uses CDN assets even if the server is configured to use local assets.  This PR changes this behaviour and will load assets from server if the server is configured to do so, ie if `HASURA_GRAPHQL_CONSOLE_ASSETS_DIR` set on the server.

### Affected components



### Related Issues
#3382

### Steps to test and verify
- Start server with `HASURA_GRAPHQL_CONSOLE_ASSETS_DIR` set to a valid directory
- Open console with `hasura console`, check if assets are loaded from server
#### Breaking changes
